settings.py

```diff
@@ -134,14 +134,18 @@ def __init__(self, multiworld_count=None):
                 description='Randomize where your starting house is located'),
             Setting('dungeonshuffle', 'Gameplay', 'u', 'Dungeon shuffle', default=False,
                 description='Randomizes the dungeon that each dungeon entrance leads to'),
-            Setting('entranceshuffle', 'Gameplay', 'E', 'Entrance randomizer', options=[("none", '', "Default"), ("simple", 's', "Simple"), ("advanced", 'a', "Advanced"), ("expert", 'E', "Expert"), ("insanity", 'I', "Insanity")], default='none',
+            Setting('entranceshuffle', 'Gameplay', 'E', 'Entrance randomizer', options=[("none", '', "Default"), ("simple", 's', "Simple"), ("split", 'S', "Split"), ("mixed", 'm', "Mixed")], default='none',
                 description="""Randomizes where overworld entrances lead to.
 [Simple] single entrance caves that contain items are randomized
-[Advanced] Connector caves are also randomized
-[Expert] Caves/houses without items are also randomized
-[Insanity] A few very annoying entrances will be randomized as well.
-If random start location and/or dungeon shuffle is enabled, then these will be shuffled with all the entrances.
-Note, some entrances can lead into water, use the warp-to-home from the save&quit menu to escape this."""),
+[Split] Connector caves are also randomized, in a separate pool from single entrance caves
+[Mixed] Connector caves are also randomized, in the same pool as single entrance caves
+If random start location and/or dungeon shuffle is enabled, then these will be shuffled with all the entrances."""),
+            Setting('shufflejunk', 'Gameplay', 'j', 'Shuffle itemless entrances', default=False,
```

Comment:
With a bunch of extra ER related settings, it might be wise that we move all ER settings to it's own category instead of in 'Gameplay'?